README.md

```diff
@@ -14,7 +14,8 @@ Add the packages to your `package.json` as usual or use `npm install --save <pac
 
 | **Name** | **Package Name** | **Current Version** | **Description** |
 |---|---|---|---|
-| Basics | `@jvalue/node-dry-basics` | `0.0.2` | *all basic functionality that need no dependencies* |
+| Basics | `@jvalue/node-dry-basics` | `0.0.3` | *all basic functionality that need no dependencies* |
+| Postgres | `@jvalue/node-dry-pg` | `0.0.1` | *postgres helpers and the pg package* |
```

Comment:
Pg's package.json says it is on version 0.0.1. 

I don't know how much effort is needed to achieve that, but using npm-version we could maybe automatically update the version numbers in the README file